data/json/mapgen/chemist holdout/chemical lab.json

```diff
@@ -0,0 +1,106 @@
+[
+{
+    "type": "mapgen",
+    "method": "json",
+    "weight": 100,

```

Comment:
If this location is supposed to have only one variant, `weight` field is redundant.

Reply:
> If this location is supposed to have only one variant, `weight` field is redundant.

That has been removed.